src/Core/Protocol.h

```diff
@@ -135,8 +137,9 @@ namespace Protocol
             KeepAlive = 6,           /// Keep the connection alive
             Scalar = 7,              /// A block of data (compressed or not).
             IgnoredPartUUIDs = 8,    /// List of unique parts ids to exclude from query processing
+            NextTaskRequest = 9,     /// String which contains an id to request a new task
```

Comment:
`ReadTaskRequest`?
prefix `Next` is not helpful :)

Reply:
Fixed.